doc/chapter-configuration.asciidoc

```diff
@@ -0,0 +1,211 @@
+outside of double quotes, and don't use single quotes for quoting — Newsboat
+doesn't support that.
+****
+
+Many of Newsboat's options expect strings as arguments, be it commands,
+passwords, dialog titles, URLs etc. Some options even take _multiple_ strings
+at once. These strings can contain spaces, which might confuse Newsboat since
+it already uses spaces to separate option names from option arguments.
+
+To help Newsboat understand your intent, put such strings into double quotes:
+
+    browser "firefox --new-tab %u"
+
+What if you need a double quote inside a string? Escape it with a backslash:
+
+    ocnews-password "UnbalancedQuotes\"AreSoFun!"
+
+And what about the backslash itself? Escape it, too! Suppose you have a program
+called `my favourite pager`, and you want to view articles with it. Newsboat
+ultimately passes commands to the shell, and shell expects spaces to be escaped
+if you want them preserved. But since Newsboat interprets backslashes, you have
+to add _another_ layer of escaping. Thus, you end up with a command like this:
+
+    pager "/usr/bin/my\\ favourite\\ pager"
+
+
+=== Shell Evaluation
+
+It is also possible to integrate the output of external commands
+into the configuration. The text between two `{backtick}` backticks is evaluated
+as shell command, and its output is used. This works like backtick evaluation in
+Bourne-compatible shells and allows users to use external information from the
+system within the configuration.
+
+=== Escaping
+
+Backticks and `+#+` characters can be escaped with a backslash
+(e.g. `{backslash}{backtick}` and `{backslash}#`).
+In this case, they are replaced with literal `{backtick}` or `+#+` in the
+configuration.
+
+=== Key Bindings
+
+You can bind a key to an operation with the <<bind-key,`bind-key`>> configuration command.
+An example keybinding looks like this:
+
+`bind-key <key> <operation> [<dialog>]`
+
+*Key*
+
+Lowercase keys, uppercase keys and special characters are written literally.
+
+Key combinations with `Ctrl` are written using the caret `^`.
+For instance `Ctrl-R` equals to `^R`.
+Please be aware that all Ctrl-related key combinations need to be written in uppercase.
+
+The following identifiers for special keys are supported:
+
+- `ENTER` (Enter key)
+- `BACKSPACE` (backspace key)
+- `LEFT` (left cursor)
+- `RIGHT` (right cursor)
+- `UP` (up cursor)
+- `DOWN` (down cursor)
+- `PPAGE` (page up cursor)
+- `NPAGE` (page down cursor)
+- `HOME` (cursor to beginning of list/article)
+- `END` (cursor to end of list/article)
+- `ESC` (Esc key)
+- `TAB` (Tab key)
+- `F1` to `F12` (F1 key to F12 key)
+
+*Operation*
+
+An operation gets executed when pressing the corresponding key.
+For a complete list of available operations see <<_newsboat_operations>> and <<_podboat_operations>>.
+
+*Dialog*
+
+A dialog is a context in which the key binding is active and is optional.
+Available dialogs are:
+
+* all (default if not specified)
```

Comment:
```suggestion
* `all` (default if not specified)
```

Ditto for all other dialog names. They're identifiers which should be put into the config verbatim; we use monospace to highlight this fact.